core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitor.scala

```diff
@@ -160,20 +172,24 @@ class AvroUpsertMonitorActor(
 
     case Some(message: PubSubMessage) =>
       //we received a message, so we will parse it and try to upsert it
-      logger.info(s"received avro upsert message: $message")
-      val (jobId, file) = parseMessage(message)
-
-      file match {
-        case "upsert.json" =>
-          logger.info(s"processing $file message for job $jobId with ackId ${message.ackId} in message: $message")
-          toFutureTry(initAvroUpsert(jobId, message.ackId)) map {
-            case Success(_) => writeResult(s"$jobId/success.json", resultString("import successful")).map(_ => ImportComplete) pipeTo self
-            case Failure(t) =>  writeResult(s"$jobId/error.json", resultString(t.getMessage)).map(_ => ImportComplete) pipeTo self
+      logger.info(s"received async upsert message: $message")
+      //post to import service pubsub topic
+      val attributes = parseMessage(message)
+      for {
+        petUserInfo <- getPetServiceAccountUserInfo(attributes.workspace, attributes.userEmail)
+        importStatus <- importServiceDAO.getImportStatus(attributes.importId, attributes.workspace, petUserInfo)
+        _ <- importStatus match {
+            case Some(ImportStatuses.ReadyForUpsert) => {
+              publishMessageToUpdateImportStatus(attributes.importId, ImportStatuses.Upserting)
```

Comment:
`publishMessageToUpdateImportStatus` calls `publishMessage` in `HttpGooglePubSubDAO` and that does a `retryWhen500OrGoogleError` [here](https://github.com/broadinstitute/rawls/blob/8b7e77cd1525bf804b1ec8705059d0e41102c011/google/src/main/scala/org/broadinstitute/dsde/rawls/google/HttpGooglePubSubDAO.scala#L85). Right now it would just error and then not do anything. If we are unable to write to the pub sub queue, what should happen?